src/services/azblob/backend.rs

```diff
@@ -70,7 +71,7 @@ use crate::Scheme;
 const X_MS_BLOB_TYPE: &str = "x-ms-blob-type";
 
 /// Builder for azblob services
-#[derive(Default, Clone)]
+#[derive(Default, Clone, Deserialize)]
```

Comment:
We don't want users to do `Deserialize` on builder.

Reply:
Fixed